server/routerlicious/.changeset/witty-years-exist.md

```diff
@@ -0,0 +1,13 @@
+---
+"@fluidframework/server-kafka-orderer": major
+"@fluidframework/server-lambdas": major
+"@fluidframework/server-memory-orderer": major
+"@fluidframework/server-routerlicious-base": major
+"@fluidframework/server-services-core": major
+"@fluidframework/server-services-ordering-kafkanode": major
+"@fluidframework/server-test-utils": major
+---
+
+`.off()` instance method added to IProducer and IOrdererConnection types and implementations
```

Comment:
Are there client implementations of these interfaces?

Reply:
Not that I can find, no.